### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by jashsayani**
See issue: https://github.com/ParsePlatform/parse-server/issues/110

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ENV APP_ID myAppId
+ENV MASTER_KEY mySecretMasterKey
+ENV FILE_KEY optionalFileKey
+
+# Expose the ports from Docker container to the VM
+EXPOSE 8080 27017 
+
+# We run index.js, but you should update this to the main
+# file defined in `main` in package.json
+CMD node index.js
```
Comment (skinp) [inline Dockerfile]: Really we should have a default working sample app.js file somewhere and use that. The image built by this Dockerfile should have a runnable parse-server container with minimal efforts.

Comment (gfosco) [inline Dockerfile]: Ah... So, the parse-server-example is that, basically... and there's a Dockerfile there.

Comment (jashsayani) [inline Dockerfile]: The Dockerfile runs index.js which is the main file. I could add a test.js that creates an instance of ParseServer.

Also, parse-server-example does not have a Dockerfile.

**Comment by gfosco** [comment]
Since this is just the router package, and not a full server, I think the Dockerfile should reside in parse-server-example.  I just merged a PR there which added it.

**Pull request closed without merging**